js/private/node-patches/fs.cjs

```diff
@@ -282,19 +284,14 @@ function patcher(roots) {
         const escapedRoot = isEscape(resolved, str);
         if (escapedRoot) {
             let next = nextHopSync(str);
-            if (!next) {
-                if (next == undefined) {
-                    // The escape from the root is not mappable back into the root; throw EINVAL
-                    throw enoent('readlink', args[0]);
-                }
-                else {
-                    // The escape from the root is not mappable back into the root; throw EINVAL
-                    throw einval('readlink', args[0]);
-                }
+            if (next === false) {
```

Comment:
This is the only functional difference AFAICT. Previously an empty string would return an error, now... idk if/when that is applicable 🤷